GNUmakefile

```diff
@@ -74,7 +75,9 @@ tools:  ## Install dev tools
 	go install github.com/terraform-linters/tflint@v0.49.0
 	go install github.com/rhysd/actionlint/cmd/actionlint@latest
 	go install golang.org/x/tools/go/analysis/passes/fieldalignment/cmd/fieldalignment@latest
+	go install github.com/vektra/mockery/v2@$(MOCKERY_VERSION)
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell go env GOPATH)/bin $(GOLANGCI_VERSION)
+	@echo "==> Note for asdf users: you may want to run: asdf reshim golang"
```

Comment:
curious: why this note?

Reply:
when using asdf, you normally need to do that when new tools are added , e.g. mockery won't work util you do that once so it's in the path.  something similar happens in python for instance.

Reply:
ok I understand, but we never mention asdf anywhere else in the repo, right? My feeling is that this is a very targeted comment for a specific type of users, but we are not targeting those users anywhere else.

Reply:
we have the file .tool-versions that is specific to asdf.   I'm recommending everybody  to use asdf and this is something easy to forget (reshim). but you're right it's specific, removing.

Reply:
removed